### PR TITLE
chore: add end of life plugin

### PR DESCRIPTION
**Pull request by dweber019**
## Hey, I just made a Pull Request!

Introducting the end of life plugin.
https://github.com/dweber019/backstage-plugin-endoflife

It will have the following functionality:
- load end of life data from endoflife.data
- load end of life data from any url
- load end of life data from the entity repository

#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [x] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [x] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: microsite/data/plugins/end-of-life.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+title: End of life
+author: dweber019
```
Comment (tudi2d) [inline microsite/data/plugins/end-of-life.yaml:3]: Optional: You can use your full name here if you prefer.

**Review by tudi2d — APPROVED** [review]
Thanks for submitting another plugin - cool idea!

File: microsite/data/plugins/end-of-life.yaml (hunk 1 above)
Comment (dweber019) [reply microsite/data/plugins/end-of-life.yaml:3]: I think I keep it like this to have it consistent any way thinking about a mono repo for all my plugins so I could change it then 😉

**Review by Rugvip — APPROVED** [review]
Nice! 👍 

Cool idea 😁

**Pull request merged**